server/views/pages/article.njk

```diff
@@ -12,9 +12,9 @@
 
 <div class="row row--cream">
   <div class="container">
-    <div class="grid">
-      <div class="grid__cell grid__cell--xl6 grid__cell--shift-xl1 grid__cell--l7 grid__cell--m6 grid__cell--shift-m1 grid__cell--s4">
-
+    <div class="grid grid--no-gutters">
```

Comment:
Not sure I understand. This is the prod article and it's inherited by the template article.